packages/onboarding/src/components/WooOnboardingTask/WooOnboardingTask.js

```diff
@@ -1,16 +1,48 @@
 /**
  * External dependencies
  */
-import { createElement } from '@wordpress/element';
+import { createElement, useEffect } from '@wordpress/element';
+import { recordEvent } from '@woocommerce/tracks';
 import { Slot, Fill } from '@wordpress/components';
 
-export const WooOnboardingTask = ( { id, ...props } ) => (
-	<Fill name={ 'woocommerce_onboarding_task_' + id } { ...props } />
-);
+export const trackView = ( taskId ) => {
+	const activePlugins = wp.data
```

Comment:
Note that these selectors are normally async, but currently we preload this data.  I think a better solution may be warranted long-term, but this is an existing issue and currently does not create any bugs.